pkg/registry/populator.go

```diff
@@ -94,6 +95,15 @@ func (i *DirectoryPopulator) globalSanityCheck(imagesToAdd []*ImageInput) error
 				errs = append(errs, PackageVersionAlreadyAddedErr{ErrorString: "Bundle already added that provides package and csv"})
 				break
 			}
+			if channel == image.annotationsFile.GetDefaultChannelName() {
+				containsDefault = true
+			}
+		}
+		if !containsDefault {
```

Comment:
historically, we've inferred the default channel when |channels|=1 -- is this happening at some other level (e.g. when getting the annotations file)? did we drop this behavior at some point?

Reply:
yep that still happens today and is what is returned by the `GetDefaultChannel` function